llvm/test/CodeGen/AMDGPU/release-vgprs-gfx12.mir

```diff
@@ -0,0 +1,356 @@
+# RUN: llc -O2 -march=amdgcn -mcpu=gfx1200 -run-pass=si-insert-waitcnts -verify-machineinstrs -o - %s | FileCheck %s -check-prefixes=CHECK,DEFAULT
+# RUN: llc -O2 -march=amdgcn -mcpu=gfx1200 -mattr=+dynamic-vgpr -run-pass=si-insert-waitcnts -verify-machineinstrs -o - %s | FileCheck %s -check-prefixes=CHECK,DVGPR
+
+--- |
```

Comment:
Is the IR section needed just for the calling conventions?

Reply:
Also for the optnone on the last function. Do we have a way to communicate that in MIR?

Reply:
No